test/test_files/zalesak_disk_godunov/zalesak_disk.i

```diff
@@ -0,0 +1,65 @@
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
```

Comment:
you'll have to name this file zalesak_disk_godunov.i to match the folder name

also add these lines to test/CTestList.cmake
add_test_re(vortex_patch_godunov)
add_test_re(zalesak_disk_godunov)

Reply:
Unused ParmParse Variables:
  [TOP]::io.line_plot_int(nvals = 1)  :: [1]
  [TOP]::amr.plt_tracer(nvals = 1)  :: [1]
  [TOP]::amrex.fpe_trap_invalid(nvals = 1)  :: [0]
  [TOP]::amr.plot_file(nvals = 1)  :: [plt]